core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -166,11 +166,15 @@ private[spark] class CoarseGrainedExecutorBackend(
         exitExecutor(1, "Received LaunchTask command but executor was null")
       } else {
         if (decommissioned) {
-          logError("Asked to launch a task while decommissioned.")
+          val msg = "Asked to launch a task while decommissioned."
+          logError(msg)
```

Comment:
I feel like I asked about this before, so sorry if it's a duplicate. Why are we changing the messages to be on seperate lines from the log statement?

Reply:
Because it is used on line 177 below. I didn't want to copy paste the string so I put it into a variable :-)

Reply:
Makes sense :)